SocketRocket/SRWebSocket.m

```diff
@@ -512,6 +512,11 @@ - (void)setupNetworkServiceType:(NSURLRequestNetworkServiceType)requestNetworkSe
         case NSURLNetworkServiceTypeVoice:
             networkServiceType = NSStreamNetworkServiceTypeVoice;
             break;
+#if TARGET_OS_IPHONE && __IPHONE_10_0
```

Comment:
This is going sound weird, but this definition is available on tvOS, even though it's not iOS 10.0.
You would need to change this to 
```
#if TARGET_OS_IPHONE && __IPHONE_OS_VERSION_MAX_ALLOWED > __IPHONE_9_3
```

Or even better:
```
#if TARGET_OS_IPHONE && __IPHONE_10_0 && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
```